nilearn/tests/test_masking.py

```diff
@@ -511,26 +531,48 @@ def test_compute_multi_epi_mask():
     assert_array_equal(mask_ab, get_data(mask_ab_))
 
 
-def test_compute_multi_gray_matter_mask():
-    pytest.raises(TypeError, compute_multi_gray_matter_mask, [])
+def test_compute_multi_brain_mask():
+    pytest.raises(TypeError, compute_multi_brain_mask, [])
 
     # Check error raised if images with different shapes are given as input
     imgs = [Nifti1Image(np.ones((9, 9, 9)), np.eye(4)),
             Nifti1Image(np.ones((9, 9, 8)), np.eye(4))]
-    pytest.raises(ValueError, compute_multi_gray_matter_mask, imgs)
+    pytest.raises(ValueError, compute_multi_brain_mask, imgs)
 
     # Check results are the same if affine is the same
     rng = np.random.RandomState(42)
     imgs1 = [Nifti1Image(rng.standard_normal(size=(9, 9, 9)), np.eye(4)),
              Nifti1Image(rng.standard_normal(size=(9, 9, 9)), np.eye(4))]
-    mask1 = compute_multi_gray_matter_mask(imgs1)
-
     imgs2 = [Nifti1Image(rng.standard_normal(size=(9, 9, 9)), np.eye(4)),
              Nifti1Image(rng.standard_normal(size=(9, 9, 9)), np.eye(4))]
-    mask2 = compute_multi_gray_matter_mask(imgs2)
 
+    # ... for whole-brain mask
+    mask1 = compute_multi_brain_mask(imgs1)
```

Comment:
here as well the test is too weak; all images are full of zeros

Reply:
Could you specify what should be done here?

Reply:
try to find examples where the loaded mask are not empty? bigger images, lower threshold maybe?
see https://github.com/nilearn/nilearn/pull/2738#discussion_r656575763

Reply:
I made also an update on this function. I basically generated the images of the test according to what you suggested [here](https://github.com/nilearn/nilearn/pull/2738#discussion_r658824814), thus they are not empty any longer.